### DISCUSSION TITLE
Import external ComicInfo.xml metadata

### DISCUSSION THREAD

**Discussion by Senn211** [gotson/komga#1641] [Ideas]
I would like to propose the possibility to import metadata through external ComicInfo.xml files in the same way we can specify local artwork assets by naming the file exactly the same as the book in the same location. For example you have a book named "TEST123.cbz", name the ComicInfo file "TEST123.xml". In the case there is an embedded file and an external one, just make whichever you believe (as the developer) makes more sense take priority. 

This would be extremely useful for me as it would allow me to update my library's metadata without having to write new archives when metadata is updated. I could just update the external xml files. 

It would also provide a way to import metadata with PDF's, and potentially open a path to exporting metadata from Komga in a safe, non-destructive manner.

**Comment by gotson** [comment]
This is not a standard used by any of the apps using `ComicInfo.xml`

**Reply by Senn211** [reply]
Respectfully, so what? It would change nothing about default behavior or how the contents of the file are read. Does external file exist? If so use that instead. But whatever, options bad I guess.

**Reply by gotson** [reply]
it's been raised in the past and closed as won't do https://github.com/gotson/komga/issues/408
